develop/TOOLS/NuGet/Consuming_NuGet.md

```diff
@@ -119,3 +119,39 @@ In case you are using a token that has an expiration date or if you entered the
 1. Select the credentials you want to edit and click *Edit* in the lower left corner.
 
 1. Update the username and/or password, and click *Save*.
+
+#### Issues with credentials
+
+In case you are having issues with the credentials in Visual Studio were each time the pop-up window for the credentials appear, you can try this way:
+
+Remove the credentials that are configured in the Credential Manager in Windows. See [Editing credentials](#editing-credentials) to find the location.
+Make sure all Visual Studio instances are closed so that there is no accidental caching going on.
+
+Go to *%appdata%\NuGet* and open the ***NuGet.config*** file. In there you'll find something similar to this:
```

Comment:
1. Remove the stored credentials from the Windows Credential Manager. You can find detailed instructions for this in the section on [Editing credentials](#editing-credentials).

2. Ensure that all instances of Visual Studio are closed to prevent accidental credential caching.

3. Navigate to the `%appdata%\NuGet` directory and locate the **NuGet.config** file. Inside this file, you will find XML content similar to the following: